src/pointer-analysis/value_set.cpp

```diff
@@ -340,7 +340,7 @@ void value_sett::get_value_set(
 {
   exprt tmp(expr);
   if(!is_simplified)
-    simplify(tmp, ns);
+    simplifier(tmp, ns);
```

Comment:
Why would one want a different simplifier? What would it look like? What are the minimal requirements?

Reply:
It simplifies an `exprt &` in place, like `simplify`. We use it to turn off the default simplifier's merging of identical struct types.